CHANGELOG.md

```diff
@@ -21,46 +21,46 @@
 
 All notable changes to this project will be documented in this file.
 
-## [Unreleased]
+## Unreleased
```

Comment:
please double checked this is all correctly merged

Reply:
Done